src/lib/css-tag.ts

```diff
@@ -9,8 +9,9 @@ part of the polymer project is also subject to an additional IP rights grant
 found at http://polymer.github.io/PATENTS.txt
 */
 
-export const supportsAdoptingStyleSheets =
-    ('adoptedStyleSheets' in Document.prototype) &&
+export const supportsAdoptingShadowStyleSheets =
```

Comment:
Why the name change? This is kind of public API, so I wouldn't...

Reply:
Sure. I've left the new semantics though, since all we care about is support on a `ShadowRoot`. There's [a polyfill for IE11](https://www.npmjs.com/package/construct-style-sheets-polyfill) which only supports `Document`; in that case, the old behavior was wrong, and would cause a crash.